x-pack/test/detection_engine_api_integration/security_and_spaces/large_prebuilt_rules_package/install_large_prebuilt_rules_package.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from 'expect';
+import { ALL_SAVED_OBJECT_INDICES } from '@kbn/core-saved-objects-server';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import { deleteAllRules, getPrebuiltRulesAndTimelinesStatus } from '../../utils';
+import { deleteAllPrebuiltRuleAssets } from '../../utils/prebuilt_rules/delete_all_prebuilt_rule_assets';
+import { installPrebuiltRulesAndTimelines } from '../../utils/prebuilt_rules/install_prebuilt_rules_and_timelines';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const es = getService('es');
+  const supertest = getService('supertest');
+  const log = getService('log');
+
+  describe('install_large_prebuilt_rules_package', () => {
+    beforeEach(async () => {
+      await deleteAllRules(supertest, log);
+      await deleteAllPrebuiltRuleAssets(es);
+    });
+
+    afterEach(async () => {
+      await deleteAllRules(supertest, log);
+      await deleteAllPrebuiltRuleAssets(es);
+    });
+
+    /**
+     * Unlike other tests that use mocks, this test uses actual rules from the
+     * package storage and checks that they are installed.
+     */
+    it('should install a package containing 15000 prebuilt rules without crashing', async () => {
+      // Verify that status is empty before package installation
+      const statusBeforePackageInstallation = await getPrebuiltRulesAndTimelinesStatus(supertest);
```

Comment:
The title says that we test an installation of a package with 15000 prebuilt rules here, but we install our regular package here. What do I miss?

Reply:
Sorry, this was missing comments in its corresponding config file. I added:
```
        /*  Tests in this directory simulate an air-gapped environment in which the instance doesn't have access to EPR.
         *  To do that, we point the Fleet url to an invalid URL, and instruct Fleet to fetch bundled packages at the
         *  location defined in BUNDLED_PACKAGE_DIR.
         *  Since we want to test the installation of a large package, we created a specific package `security_detection_engine-100.0.0`
         *  which contains 15000 rules assets, and attempt to install it.
         */
```
The config file points Fleet to a `BUNDLED_PACKAGE_DIR` where we created and uploaded the large package with 15000 rule assets. When we try to install, it will install that package from the zip file.